cluster/gce/manifests/kube-addon-manager.yaml

```diff
@@ -30,13 +30,21 @@ spec:
     - mountPath: /var/log
       name: varlog
       readOnly: false
+    - mountPath: /etc/srv/kubernetes/addon-manager/
+      name: srvkube
+      readOnly: true
     env:
     - name: KUBECTL_EXTRA_PRUNE_WHITELIST
       value: {{kubectl_extra_prune_whitelist}}
+    - name: KUBECTL_OPTS
+      value: '--kubeconfig=/etc/srv/kubernetes/addon-manager/kubeconfig'
```

Comment:
Does these apply to https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/resources/manifests/kube-addon-manager.yaml as well?

Reply:
I think they do, added and am testing.